SECURITY.md

```diff
@@ -410,7 +410,7 @@ grpc-netty version | netty-handler version | netty-tcnative-boringssl-static ver
 1.32.x-1.34.x      | 4.1.51.Final          | 2.0.31.Final
 1.35.x-1.41.x      | 4.1.52.Final          | 2.0.34.Final
 1.42.x-1.43.x      | 4.1.63.Final          | 2.0.38.Final
-1.44.x             | 4.1.72.Final          | 2.0.46.Final
+1.44.x-            | 4.1.72.Final          | 2.0.46.Final
```

Comment:
Do you want to claridy what `-` means? Maybe something like
```suggestion
1.44.x-current     | 4.1.72.Final          | 2.0.46.Final
```

Reply:
Meh. It seems fine either way. I wasn't really trying to change anything or trying to think about what way to do it. I'm just fixing it as it was broken in https://github.com/grpc/grpc-java/commit/d7f951a9d8a259682b786b711cd7431834a283cd#diff-f6ed156e4bf5c791680662464b94ea5d753f219ee816b385f67870e2c0d7d4c7 . The open version range has been used since this table was introduced, and it hasn't seemed to be a problem.